projects/experimental/directives/button-group/button-group.style.less

```diff
@@ -0,0 +1,68 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButtonGroup] {
+    display: flex;
+    border-radius: var(--tui-radius-xl);
+    justify-content: center;
+    overflow: hidden;
+
+    > button {
```

Comment:
Let's apply same styles for `a` as well, maybe they will be links, not buttons sometimes.

Reply:
fixed